.circleci/circle.yml

```diff
@@ -0,0 +1,20 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/app
+    docker:
+      - image: circleci/node:8
+    steps:
+      - checkout
+      - restore_cache:
+          key: dependency-cache-{{ checksum "package-lock.json" }}
+      - run:
+          name: npm-install
+          command: npm install
+      - save_cache:
+          key: dependency-cache-{{ checksum "package-lock.json" }}
+          paths:
+            - .node_modules
+      - run:
+          name: test
+          command: npm test
```

Comment:
you should add a deploy step to circleci that builds the docker image and uploads it, like this:
<pre>
deployment:
  latest:
    branch: master
    commands:
    - docker login -e $DOCKER_EMAIL -u $DOCKER_USER -p $DOCKER_PASS
    - docker build . -t mozilla/portal:latest
    - docker push mozilla/portal:latest
</pre>
I will add the needed env vars in circle when this is ready